organization-bootstrap/environments/README.md

```diff
@@ -0,0 +1,55 @@
+# Environment-based organizational sample
+
+This sample creates an organizational layout with a single level, where each  folder is usually mapped to one infrastructure environment (test, dev, etc.). It also sets up all prerequisites for automation (GCS state buckets, service accounts, etc.), and the correct roles on those to enforce separation of duties at the environment level.
+
+This layout is well suited for medium-sized infrastructures managed by a small set of teams, where the complexity in application resource ownership and access roles is mostly dealt with at the project level, and/or in the individual services (GKE, Cloud SQL, etc.). Its simplicity also makes it a good starting point for more complex or specialized layouts.
+
+![High-level diagram](diagram.png "High-level diagram")
+
+Refer to the [section-level README](../README.md) for general considerations about this type of samples, and usage instructions.
+
+## Managed resources and services
+
+This sample creates several distinct groups of resources:
+
+- one folder per environment
+- one top-level project to hold Terraform-related resources
```

Comment:
I'd encourage putting these shared services into a "shared" folder.

Reply:
All the actual examples I've had so far of using this with customers have only ever used a single project for shared resources, and having it at the top level makes it very clear that it's not tied to a single environment (or a single business unit in the 2-level example we'll add soon).
That said, I can see the merit of having a Shared folder if multiple shared projects are needed. My preference would be to leave as is as it reflects actual installs and is simpler, and add a comment in `main.tf` to suggest using a folder for more complex needs.